doc/enterprise/deployment.md

```diff
@@ -0,0 +1,117 @@
+# Deploying Enterprise Edition
+
+Most of the Pachyderm Enterprise features are wrapped up in our "dashboard" currently (other than a few that are currently under active development).  Thus, Pachyderm Enterprise deployment is as simple as deploying the dashboard on top of or along with a Pachyderm cluster:
```

Comment:
Remove the "currently" caveat. Instead of "most...wrapped up in dashboard" --> "The primary interface for the Enterprise Edition is the Pachyderm dashboard." No "thus" needed after that.